src/types.ts

```diff
@@ -11,7 +11,8 @@ export interface RHDHThemePalette {
     mainSectionBackgroundColor: string;
     headerBottomBorderColor: string;
     cardBackgroundColor: string;
-    sideBarBackgroundColor: string;
+    sidebarBackgroundColor: string;
```

Comment:
For my ask above, should we maybe mark this as deprecated instead?

```suggestion
    /** @deprecated please use `sidebarBackgroundColor` instead */
    sideBarBackgroundColor: string;
    sidebarBackgroundColor: string;
```